src/ui/components/CredCardTemplate/CredCardTemplate.tsx

```diff
@@ -5,19 +5,10 @@ import { Alert } from "../Alert";
 import { CredCardTemplateProps } from "./CredCardTemplate.types";
 import { CredentialMetadataRecordStatus } from "../../../core/agent/records/credentialMetadataRecord.types";
 import { i18n } from "../../../i18n";
-import {
-  ConnectionType,
-  CredentialType,
-} from "../../../core/agent/agent.types";
 import CardBodyPending from "./CardBodyPending";
 import CardBodyGeneric from "./CardBodyGeneric";
```

Comment:
"CardBodyGeneric" was initially built to provide all non-custom cards with a plain W3C card template (the old blue background). If we are just going with Keri I believe this could be renamed and refactored. No need to change it right now if you don't feel to @bao-sotatek we can leave this not for later so we'll remember, but please look further down as some of the old W3C naming is still present in the classnames, maybe just rename those for the moment.